docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>I don't think this is necessary: we want this tutorial to be as concise as possible and shouldn't try to explain things unrelated to TensorBoard.dev too much. If we want to keep it, perhaps the more concise: "Use the MNIST dataset and create a simple TensorFlow model."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done. I removed this section.